src/features/devices/hooks/should-be-managed-by--release.ts

```diff
@@ -0,0 +1,136 @@
+import { hooks, sbvrUtils, permissions } from '@balena/pinejs';
```

Comment:
Can you move this to a supervisorapp feature equivalent to the hostapp feature, cc @xginn8

Reply:
Done by Matt